<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Footjob'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Handjob'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_ShortTerm_Sex_AdditionalSexScore'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_ShortTer' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_ShortTer' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelTrack_StripClub_Charm'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_StripClub_Charm' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_StripClub_Charm' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Impregnated_SimB'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelTrack_Personality_Archetype_Impression'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Personality_Arche' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Personality_Arche' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_NoProtection'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Oraljob'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Anal'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Impregnated_SimA'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelationshipTrack_Memories_MemoryType_Sex_Vaginal'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelationshipTrack_Memories' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 13:55:15</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>bitless_relationship_track.py:22</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[manus] Exception in &lt;class 'sims4.tuning.instances.TURBODRIVER:WickedWhims_RelTrack_Relationship_WhiteRomance'&gt;._tuning_loaded_callback. (AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Relationship_Whit' has no attribute '_initialize_decay_override_list')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 167, in c_api_server_init_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\core_services.py", line 164, in start_service_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\service_manager.py", line 346, in start_single_service&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 158, in update_incremental_start&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 217, in _execute_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\tuning\instance_manager.py", line 616, in invoke_registered_callbacks_gen&#13;&#10;  File ".\WickedWhims_v178c\turbolib2\gizmos\bitless_relationship_track.py", line 22, in _tuning_loaded_callback&#13;&#10;AttributeError: type object 'TURBODRIVER:WickedWhims_RelTrack_Relationship_Whit' has no attribute '_initialize_decay_override_list'rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>fbf12e178806fa2367e0f357</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2025-03-24 16:01:46</createtime><buildsignature>Local.Unknown.Unknown.1.108.335.1020-1.300.000.188.Release</buildsignature><categoryid>posture_graph.py:5164</categoryid><desyncid>fbf12e178806fa2367e0f357</desyncid><systemconfig/><screenshot/><desyncdata>[rez] Posture graph threw an exception while being queried by autonomy: (AssertionError: Source spec not in the posture graph: stand@object_paintEaselGEN_01:0x0fe4042667a30000, carry, surface:slot_target@object_paintEaselGEN_01:0x0fe4042667a30000 for interaction:  running canvas_PaintPainting_Staging_Large:2196 on object_Canvas_large:0x09b81463992f05f9  (guaranteed))&#13;&#10;Traceback (most recent call last):&#13;&#10;Caught and logged:&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 2988, in _get_transitions_for_sim&#13;&#10;  File "postures\posture_graph.py", line 5164, in postures.posture_graph.PostureGraphService.get_segmented_paths&#13;&#10;AssertionError: Source spec not in the posture graph: stand@object_paintEaselGEN_01:0x0fe4042667a30000, carry, surface:slot_target@object_paintEaselGEN_01:0x0fe4042667a30000 for interaction:  running canvas_PaintPainting_Staging_Large:2196 on object_Canvas_large:0x09b81463992f05f9  (guaranteed)rtim=0&#13;&#10;Client Session Time: 29.91&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0xdd4141d5&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 12,168,0,5425&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19045 GPU: NVIDIA GeForce GTX 1080  32.0.15.7260, GUID: 31A6DF6B-79F8-11EE-8422-00E04C5662AD&#13; &#10;</desyncdata></report>
</root>